nltk/metrics/agreement.py

```diff
@@ -50,10 +50,10 @@
     >>> t = AnnotationTask(data=[x.split() for x in open(os.path.join(os.path.dirname(__file__), "artstein_poesio_example.txt"))])
     >>> t.avg_Ao()
     0.88
-    >>> t.pi()
-    0.7995322418977615...
-    >>> t.S()
-    0.8199999999999998...
+    >>> round(t.pi(), 5)
+    0.79953
+    >>> round(t.S(), 3)
+    0.82
```

Comment:
This seems odd, rounding to 3 gives 2 values after the dot?

Reply:
Yes, because if not rounded we have 0.8199999999999998 :P

Reply:
Oooh, that's rough, then `0.82...` wouldn't work either (presumably). Rounding is acceptable then, otherwise the output (e.g. `0.819...`) may change in the future or on different devices (e.g. to `0.820...`).

Reply:
Wouldn't _round(t.S(), 2)_ be more appropriate, then?